module/Application/view/layout/layout.phtml

```diff
@@ -74,26 +73,58 @@ function getIsActiveClass($view, array $condition)
-                                    <span class="visible-lg-inline"><?= $this->translate('Signed in as') ?> </span><?= $member->getFirstName() ?>
-                                    <span class="caret"></span>
-                                </a>
-                                <ul class="dropdown-menu">
-                                    <li><a href="<?= $this->url('member') ?>"><?= $this->translate('My information') ?></a></li>
-                                    <li><a href="#">SuSOS</a></li>
-                                    <li><a href="<?= $this->url('user/default', array(
-                                        'action' => 'logout'
-                                    )) ?>"><?= $this->translate('Logout') ?></a></li>
-                                </ul>
-                            </li>
-                        <?php endif; ?>
+                    </ul>
+                    <ul class="nav navbar-nav navbar-right">
+                        <li>
+                            <a href="#">EN</a>
+                        </li>
+                        <ul class="nav navbar-nav navbar-right login-menu">
+                            <?php if (null === $this->identity()): ?>
+                                <li class="dropdown"><a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">
+                                            <?= $this->translate('Login') ?>
+                                            <b class="caret"></b></a>
+                                            <ul class="dropdown-menu">
+                                                <form action="<?= $this->url('user') ?>" method="post">
+                                                    <div class="form-group">
+                                                        <input type="text" name="login" placeholder="<?= $this->translate('Member ID or email address') ?>" class="form-control">
+                                                    </div>
+                                                    <div class="form-group">
+                                                        <input type="password" name="password" placeholder="<?= $this->translate('Password') ?>" class="form-control">
+                                                    </div>
+                                                    <div class="checkbox">
+                                                        <label>
+                                                            <input type="checkbox" value="1">
+                                                            <?= $this->translate('Stay logged in') ?>
+                                                        </label>
+                                                    </div>
+                                                    <li>
+                                                        <input type="submit" value="<?= $this->translate('Login') ?>" class="btn btn-primary form-control">
```

Comment:
Actually something like this probably is easier: https://gist.github.com/jszanto/90017eb8f594fc6adf93